src/platform/backend-device-group.h

```diff
@@ -0,0 +1,135 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2015 Intel Corporation. All Rights Reserved.
```

Comment:
Consider updating to 2023, it's a new file

Reply:
Will fix.